packages/synapse-react-client/src/components/FullTextSearch.tsx

```diff
@@ -73,13 +67,7 @@ export const FullTextSearch: React.FunctionComponent<FullTextSearchProps> = ({
   }
 
   return (
-    <div
-      className={`QueryWrapperSearchInput ${
-        showFacetFilter
-          ? QUERY_FILTERS_EXPANDED_CSS
-          : QUERY_FILTERS_COLLAPSED_CSS
-      }`}
```

Comment:
remove this check from every component, except QueryWrapperPlotNav. In `QueryWrapperPlotNav`, we apply the class once and update selectors accordingly.